wpeverest/user-registration
# UR-378 Fix - Required field not working when conditional logic enabled.

### All Submissions:






### Changes proposed in this Pull Request:

Previously, When we apply conditional logic to any required field and in frontend if we bypass the  client side required validation from browser console. then it was submitting the form instead server side validation. This issue has been fixed in this PR.

### How to test the changes in this Pull Request:

1. Create a form with required field and conditional logic enabled.
2. Goto frontend Form Show that required field and goto inspect element and remove the **required="required"**.
3. Submit the form without entering any value in that field. It should validate from server side.

### Types of changes:




### Other information:




### Changelog entry

>  Fix - Required field not working when conditional logic enabled.
